src/Containers/FileUpload/FileUpload.tsx

```diff
@@ -35,6 +33,9 @@ interface IState{
     isDragEnter:boolean;
 }
 
+const padding=10;
+const margin=10;
```

Comment:
PADDING
MARGIN
declare outside component.

Reply:
just one thing, in the javascript style guide of the docs it says camelCase is permitted when declaring deeply immutable constants.

Reply:
https://wolfbeacon.atlassian.net/wiki/spaces/CW/pages/1130102818/JavaScript+Style+Guide#2.4.-Use-Upper-Case-for-Deeply-Immutable-Constants

I would go with the good option rather than permitted if I have a choice?

Reply:
in fact in the docs says permitted for uppercase and not required to use uppercase. I think it's a matter of taste